pkg/frontend/openshiftversions_list.go

```diff
@@ -22,38 +21,29 @@ func (f *frontend) listInstallVersions(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	log := ctx.Value(middleware.ContextKeyLog).(*logrus.Entry)
 
+	log.Printf("here")
```

Comment:
rm this

Reply:
that's embarrassing... ty for catching :)

Reply:
removed